controllers/vrg_volrep.go

```diff
@@ -672,16 +672,15 @@ func (v *VRGInstance) reconcileVRForDeletion(pvc *corev1.PersistentVolumeClaim,
 		case requeueResult:
 			return requeue
 		case !ready:
-			// Ensure VR is available at the required state before deletion
-			return !requeue
+			return requeue
```

Comment:
This should have returned `requeue` but was working as the caller just falls through post calling this function in the for loop, and is corrected by looking at the tests and log output from the same.